lib/core/src/lib/datatable/components/amount-cell/amount-cell.component.ts

```diff
@@ -0,0 +1,66 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+    ChangeDetectionStrategy,
+    Component,
+    ViewEncapsulation,
+    Input,
+    Optional,
+    OnInit,
+    DEFAULT_CURRENCY_CODE,
+    Inject
+} from '@angular/core';
+import { DataTableCellComponent } from '../datatable-cell/datatable-cell.component';
+import { DataTableService } from '../../services/datatable.service';
+import { CurrencyConfig } from '../../data/data-column.model';
+import { CommonModule } from '@angular/common';
+
+@Component({
+    standalone: true,
+    imports: [CommonModule],
+    selector: 'adf-amount-cell',
+    templateUrl: './amount-cell.component.html',
+    host: { class: 'adf-datatable-content-cell' },
```

Comment:
WDYT about add another class to the host 
```suggestion
    host: { class: 'adf-amount-cell adf-datatable-content-cell' },
```

Reply:
as far as i know, we don't need to have additional styles for this component, correct me if im wrong

Reply:
not but I notice this pattern in the current cell, maybe it's better for testing instead of querying 'span' we can query 'adf-amount-cell-value' (according to the previous comment)

Reply:
i don't think is a good practice to use fake class only for test purposes. in that component we have only one element, so it's no problem to query it as a span. btw, from e2e point of view, we have a data automation id added at the datatable component level, which contains type of widget (amount) + value, so for each row we have a different values. from this point, there is no benefit to add it also here if it is the point (testing)
<img width="855" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/49343696/0a1d027f-6b30-4170-9806-a65b84d315e3">

Reply:
Ok, agree